decisions/0009-schema-refactor.md

```diff
@@ -0,0 +1,95 @@
+---
+status: accepted 
+date: 2023-12-19
+deciders: Chris Mungall, Lee Ann 
```

Comment:
All the other names are "first last", but one of these is "first" only.

Reply:
Fixed, will be in push